dcspy/migration.py

```diff
@@ -54,6 +55,20 @@ def _filter_api_ver_func(cfg_ver: str) -> Iterator[Callable[[DcspyConfigYaml], N
             yield globals()['_api_ver_{}'.format(api_ver.replace('.', '_'))]
 
 
+def _api_ver_3_2_0(cfg: DcspyConfigYaml) -> None:
+    """
+    Migrate to version 3.2.0.
+
+    :param cfg: Configuration dictionary
+    """
+    user_appdata = get_config_yaml_location()
+    makedirs(name=user_appdata, exist_ok=True)
+    pattern = re.compile(r'(\S+):\s*(\S+)\sCUSTOM\s+(\S+)\s+\d\s*\|\s*(\S+)\s*\d\s*\|')
```

Comment:
This should only replace custom entries that have a `1` and then a `0`. A push button will do the `1` and `0`, but if there is a custom that does `0` then `1` (or even `2` then `1`) then changing this to push button will break that. For example, see comment further up re: the rocker switch in the Apache `PLT_EUFD_RTS`.

Reply:
Yes, Good catch! I will update reqexp.